components/ui/Navbar.js

```diff
@@ -1,32 +1,59 @@
-"use client"
-import classNames from 'classnames';
-import { usePathname } from 'next/navigation';
-import Link from 'next/link'
+"use client";
+import classNames from "classnames";
+import { usePathname } from "next/navigation";
+import { useState } from "react";
+import Link from "next/link";
 
-import styles from './Navbar.module.css';
+import styles from "./Navbar.module.css";
 
-const navbarItems = [
-  {
-    title: 'ABOUT US',
-    link: '/about_us',
-  },
-  {
-    title: 'DESTINATION',
-    link: '/destination',
-  },
-  {
-    title: 'NASA COLLABORATION',
-    link: '/nasa_collaboration',
-  }
-];
+const NavItem = ({ title, link, isActive, onClick }) => {
+  return (
+    <li>
+      <a
+        href={link}
+        onClick={onClick}
+        style={{
+          color: isActive ? "grey" : "white",
+          textDecoration: isActive ? "underline" : "none",
+          padding: "10px",
+        }}
+      >
+        {title}
+      </a>
+    </li>
+  );
+};
+
+export default NavItem;
 
 export const Navbar = () => {
-  const currentPath = usePathname()
+  const [activeLink, setActiveLink] = useState();
+
+  const NavBarItems = [
```

Comment:
```suggestion
  const navBarItems = [
```
PascalCase is reserved for components in React.